packages/web/app/src/router.tsx

```diff
@@ -80,9 +79,8 @@ const LazyTanStackRouterDevtools = lazy(() =>
 );
 
 function identifyOnSentry(userId: string, email: string): void {
-  configureScope(scope => {
-    scope.setUser({ id: userId, email });
-  });
+  const scope = getCurrentScope();
+  scope.setUser({ id: userId, email });
```

Comment:
Let's not do any Sentry related changes in this PR. We need to migrate to newer Sentry anyway, but not here.